DSCResources/MSFT_xDnsARecord/MSFT_xDnsARecord.psm1

```diff
@@ -0,0 +1,73 @@
+﻿function Get-TargetResource
```

Comment:
All DSC modules in the DscResources should have tests written using Pester included in a Tests folder.
Please include tests for the new code you added:
[Contribution Guidelines](https://github.com/PowerShell/DscResources/blob/master/CONTRIBUTING.md#improving-test-coverage-for-existing-resources)